docs/source/projects.rst

```diff
@@ -336,6 +336,9 @@ In addition, **signac** also provides the :py:meth:`signac.Project.fn` method, w
     >>> print(project.fn('foo.bar'))
     '/home/johndoe/my_project/foo.bar'
 
+
+Be careful when accessing the project-level data concurrently from different running jobs, as the underlying HDF5 file is locked by default, even when data is only being read. If data will only be read, the environment variable ``HDF5_USE_FILE_LOCKING`` can be set to ``FALSE`` to avoid this behaviour. For concurrent writing and reading, you can check |h5py_swmr|_, |multiprocessing_sync|_ or |posix_ipc_semaphores|_.
```

Comment:
It probably makes sense to be more specific about what may happen when you try to access the project-level data concurrently from different running jobs, so that users can more easily find this bit of advice in the docs. Will it always raise a `OSError`? If so, I would add `OSError` somewhere so that a search on the page will take you here.

Reply:
Solved in 205aa57 , thanks for the suggestion!